packages/components/src/components/step/step.ts

```diff
@@ -0,0 +1,176 @@
+import { css, unsafeCSS } from 'lit';
+import { customElement } from '../../internal/register-custom-element';
+import { html, literal } from 'lit/static-html.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { property } from 'lit/decorators.js';
+import componentStyles from '../../styles/component.styles';
+import cx from 'classix';
+import ParagraphStyles from '../../styles/paragraph/paragraph.css?inline';
+import SolidElement from '../../internal/solid-element';
+/**
+ * @summary Steps are used inside [step groups](/components/step-group) to guide users through the steps of a process or task..
+ * @documentation https://solid.union-investment.com/[storybook-link]/step
+ * @status stable
+ * @since 3.2.0
+ *
+ * @dependency sd-icon
+ * @dependency sd-divider
+ *
+ * @slot - The step's description.
+ * @slot label - The step's label.
+ *
+ *
+ * @csspart base - The component's base wrapper.
+ * @csspart circle-and-tail-container - The container that wraps the step's circle and tail.
+ * @csspart circle - The circle that marks the step's state.
+ * @csspart tail - The step's tail.
+ * @csspart text-container - The container that wraps the step's label and description.
+ * @csspart label - The step's label.
+ * @csspart description - The step's description.
+ */
+@customElement('sd-step')
+export default class SdStep extends SolidElement {
+  /** The step's size. */
+  @property({ reflect: true }) size: 'lg' | 'sm' = 'lg';
+
+  /** Determines the orientation of the step. */
+  @property({ reflect: true }) orientation: 'horizontal' | 'vertical' = 'horizontal';
+
+  /** Determines the state of the step. */
+  @property({ reflect: true }) state: 'waiting' | 'finished' | 'inProgress' = 'waiting';
```

Comment:
Can we changed the order to => 'waiting' | 'inProgress' | 'finished' = 'waiting'; ?